packages/@lwc/engine-core/src/framework/renderer.ts

```diff
@@ -13,7 +13,7 @@ export interface Renderer<N = HostNode, E = HostElement> {
     syntheticShadow: boolean;
     insert(node: N, parent: E, anchor: N | null): void;
     remove(node: N, parent: E): void;
-    createElement(tagName: string, namespace?: string): E;
+    createElement(tagName: string, isSvg?: boolean): E;
```

Comment:
As an alternative, consider adding a `createElementNS(ns, tagName)` method to Renderer interface. This will accommodate for other tags that need to be created with a namespaceURI. IMO, we should keep the renderer interface as generic as possible.

Reply:
Good point let me address this.